sea-rt/seahorn.cpp

```diff
@@ -29,10 +29,23 @@ void __VERIFIER_assume(int x) {
   assert(x);
 }
 
+void __VERIFIER_assert(bool x, int property) {
+  sealog("[sea] __VERIFIER_assert was called for property: %d,\n", property);
+  assert(x);
+}
+
 void __SEA_assume(bool x) {
   assert(x);
 }
 
+void btor2mlir_print_input_num(unsigned num, unsigned value, unsigned width) { 
```

Comment:
this definitely seems btor2mlir related and not seahorn related. 

A better solution is to move these functions into btor2mlir and provide them as extra library to seahorn during cex generation. 

exe cex command should be able to take multiple files and link them together. 

you are relying on seahorn linking it's rt by default. But this is not the right way to link desired code in

Reply:
Sounds good. I'll do that instead.